TabulateSmarterTestContentPackage/Extractors/ItemStandardExtractor.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Xml;
+using System.Xml.Linq;
+using System.Xml.XPath;
+using TabulateSmarterTestContentPackage.Models;
+
+namespace TabulateSmarterTestContentPackage.Extractors
+{
+    public static class ItemStandardExtractor
+    {
+        public static IEnumerable<ItemStandard> Extract(XElement metadata, string standard = "PrimaryStandard")
+        {
+            var sXmlNs = new XmlNamespaceManager(new NameTable());
+            sXmlNs.AddNamespace("sa", "http://www.smarterapp.org/ns/1/assessment_item_metadata");
+            var xmlNodes = metadata.XPathSelectElements($".//sa:{standard}", sXmlNs).ToList();
+            if (!xmlNodes.Any())
+            {
+                return new List<ItemStandard>();
+            }
+            var result = xmlNodes.Select(x => new
+                {
+                    Standard = x.Value.Split(':').FirstOrDefault(),
+                    Metadata = x.Value.Split(':').LastOrDefault()?.Split('|')
+                })
+                .Select(x => new ItemStandard
+                {
+                    Standard = x.Standard ?? string.Empty,
+                    Claim = x.Metadata.FirstOrDefault() ?? string.Empty,
+                    Target =
+                        x.Metadata.Skip(SkipToTargetForPublication(x.Standard ?? string.Empty))
+                            .FirstOrDefault() ?? string.Empty,
+                    ContentDomain = x.Metadata.Skip(SkipToContentDomainForPublication(x.Standard ?? string.Empty))
+                                        .FirstOrDefault() ?? string.Empty
+                }).ToList();
+            if (result.Count > 1 && standard.Equals("PrimaryStandard"))
+            {
+                return DeterminePrimaryStandard(result).ToList();
+            }
+            return result;
+        }
+
+        // Functionally, there are a maximum of two primary standards. If we have the v6 first and reverse we should get the good one
+        private static IEnumerable<ItemStandard> DeterminePrimaryStandard(IList<ItemStandard> candidates)
+        {
+            return candidates.Count() > 1 && candidates.First().Standard.Equals("SBAC-MA-v6")
```

Comment:
`candidates.Any()` or could with C#6 can't you do something like this: `candidates.FirstOrDefault()?.Standard.Equals("SBAC-MA-v6")`